File: generated-sources/scala-gatling/mojang-api/src/gatling/scala/org/openapitools/client/model/SecurityAnswerId.scala

package org.openapitools.client.model


case class SecurityAnswerId (
    /* A number giver by Mojang to reply a question */
    _id: Integer
)
object SecurityAnswerId {
    def toStringBody(var_id: Object) =
        s"""
        | {
        | "id":$var_id
        | }
        """.stripMargin
}

File: generated-sources/php-ze-ph/mojang-api/src/App/DTO/SecurityAnswerId.php
<?php

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 * A security question answer id given by Mojang
 */
class SecurityAnswerId
{
    /**
     * A number giver by Mojang to reply a question
     * @DTA\Data(field="id")
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $id;
}
